[Bug] 安卓9打包后闪退

### Steps to reproduce/复现步骤

1. 使用jar打包
2. 安装在安卓9设备上
3. 启动软件

### Expected behaviour/预期行为

正常启动

### Actual behaviour/实际行为

闪退

### Xposed Module List/Xposed 模块列表

```Shell
N/A
```


### LSPatch version/LSPatch 版本

0.4

### Android version/Android 版本

9

### Shizuku version/Shizuku 版本

N/A

### Version requirement/版本要求

- [X] I am using latest debug CI version of LSPatch and enable verbose log/我正在使用最新 CI 调试版本且启用详细日志

### Logs/日志
```
I/LSPatch: Use manager: false
I/LSPatch: Signature bypass level: 0
I/LSPatch: hooked app initialized: android.app.LoadedApk@81c97e5
W/LSPatch: Original AppComponentFactory not found: androidx.core.app.CoreComponentFactory
D/LSPatch: Initialize service client
I/LSPatch: Bootstrap Xposed
I/LSPatch: Load modules
I/LSPatch: Modules initialized
I/LSPosed-Bridge: Crash unexpectedly: java.lang.ExceptionInInitializerError
        at org.lsposed.lspatch.appstub.LSPAppComponentFactoryStub.<clinit>(Unknown Source:168)
        at java.lang.Class.newInstance(Native Method)
        at android.app.LoadedApk.createAppFactory(LoadedApk.java:227)
        at android.app.LoadedApk.createOrUpdateClassLoaderLocked(LoadedApk.java:731)
        at android.app.LoadedApk.getClassLoader(LoadedApk.java:810)
        at android.app.LoadedApk.getResources(LoadedApk.java:1032)
        at android.app.ContextImpl.createAppContext(ContextImpl.java:2345)
        at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5798)
        at android.app.ActivityThread.access$1100(ActivityThread.java:200)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1656)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6718)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
     Caused by: java.lang.RuntimeException: Do hook
        at org.lsposed.lspatch.loader.LSPApplication.onLoad(LSPApplication.java:110)
        at java.lang.Runtime.nativeLoad(Native Method)
        at java.lang.Runtime.load0(Runtime.java:926)
        at java.lang.System.load(System.java:1633)
        at org.lsposed.lspatch.appstub.LSPAppComponentFactoryStub.<clinit>(Unknown Source:161)
        at java.lang.Class.newInstance(Native Method) 
        at android.app.LoadedApk.createAppFactory(LoadedApk.java:227) 
        at android.app.LoadedApk.createOrUpdateClassLoaderLocked(LoadedApk.java:731) 
        at android.app.LoadedApk.getClassLoader(LoadedApk.java:810) 
        at android.app.LoadedApk.getResources(LoadedApk.java:1032) 
        at android.app.ContextImpl.createAppContext(ContextImpl.java:2345) 
        at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5798) 
        at android.app.ActivityThread.access$1100(ActivityThread.java:200) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1656) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loop(Looper.java:193) 
        at android.app.ActivityThread.main(ActivityThread.java:6718) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858) 
     Caused by: java.lang.NoSuchFieldError: android.app.LoadedApk#mDefaultClassLoader
        at de.robv.android.xposed.XposedHelpers.lambda$findField$1(XposedHelpers.java:246)
        at de.robv.android.xposed.XposedHelpers$$ExternalSyntheticLambda3.get(Unknown Source:2)
        at java.util.Optional.orElseThrow(Optional.java:286)
        at de.robv.android.xposed.XposedHelpers.findField(XposedHelpers.java:246)
        at de.robv.android.xposed.XposedHelpers.getObjectField(XposedHelpers.java:1028)
        at org.lsposed.lspatch.loader.LSPApplication.switchClassLoader(LSPApplication.java:328)
        at org.lsposed.lspatch.loader.LSPApplication.onLoad(LSPApplication.java:106)
        at java.lang.Runtime.nativeLoad(Native Method) 
        at java.lang.Runtime.load0(Runtime.java:926) 
        at java.lang.System.load(System.java:1633) 
        at org.lsposed.lspatch.appstub.LSPAppComponentFactoryStub.<clinit>(Unknown Source:161) 
        at java.lang.Class.newInstance(Native Method) 
        at android.app.LoadedApk.createAppFactory(LoadedApk.java:227) 
        at android.app.LoadedApk.createOrUpdateClassLoaderLocked(LoadedApk.java:731) 
        at android.app.LoadedApk.getClassLoader(LoadedApk.java:810) 
        at android.app.LoadedApk.getResources(LoadedApk.java:1032) 
        at android.app.ContextImpl.createAppContext(ContextImpl.java:2345) 
        at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5798) 
        at android.app.ActivityThread.access$1100(ActivityThread.java:200) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1656) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loop(Looper.java:193) 
        at android.app.ActivityThread.main(ActivityThread.java:6718) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858) 

```

--- a/patch-loader/src/main/java/org/lsposed/lspatch/loader/LSPApplication.java
+++ b/patch-loader/src/main/java/org/lsposed/lspatch/loader/LSPApplication.java
@@ -102,9 +102,7 @@ public class LSPApplication {
             LSPLoader.initModules(appLoadedApk);
             Log.i(TAG, "Modules initialized");
 
-            switchClassLoader("mBaseClassLoader");
-            switchClassLoader("mDefaultClassLoader");
-            switchClassLoader("mClassLoader");
+            switchAllClassLoader();
             doSigBypass(context);
         } catch (Throwable e) {
             throw new RuntimeException("Do hook", e);
@@ -324,8 +322,13 @@ public class LSPApplication {
         }
     }
 
-    private static void switchClassLoader(String fieldName) {
-        var obj = XposedHelpers.getObjectField(appLoadedApk, fieldName);
-        XposedHelpers.setObjectField(stubLoadedApk, fieldName, obj);
+    private static void switchAllClassLoader() {
+        var fields = LoadedApk.class.getDeclaredFields();
+        for (Field field : fields) {
+            if (field.getType() == ClassLoader.class) {
+                var obj = XposedHelpers.getObjectField(appLoadedApk, field.getName());
+                XposedHelpers.setObjectField(stubLoadedApk, field.getName(), obj);
+            }
+        }
     }
 }

